packages/fastify-react/plugin/virtual.js

```diff
@@ -42,19 +56,34 @@ export async function resolveId (id) {
 
 export function loadVirtualModule (virtualInput) {
   let virtual = virtualInput
```

Comment:
Instead of making two arrays, maybe it's easier to split the `virtualInput` into its filename and extName? 

```javascript
const extension = path.extname(virtualInput)
const filename = path.basename(virtualinput, extension)

if (!virtualModules.has(filename) {
  return
}
```